src/modules/user_certificate/user_certificate.service..ts

```
@@ -0,0 +1,262 @@
+import { HttpStatus, Injectable } from '@nestjs/common';
+import { Response } from 'express';
+
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { UserCourseCertificate } from '../certificate/entities/user_course_certificate';
+import { CreateCertificateDto } from './dto/create-user-certificate-dto';
+import APIResponse from 'src/common/utils/response';
+import { LoggerService } from 'src/common/logger/logger.service';
+import { ConfigService } from '@nestjs/config';
+const axios = require('axios');
```

ESLint

[error] 'axios' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)